packages/oui-field/src/index.spec.js

```diff
@@ -466,6 +520,40 @@ describe("ouiField", () => {
             });
         });
 
+        describe("with validation", () => {
```

Comment:
Good test for custom message!
We should test default error messages too. Almost identical to this one

Reply:
Yeah, but it the test should not enforce a string but verify it's actually using the one from the provider

Reply:
Yes I agree, provider should be mocked and we should only "verify it's actually using the one from the provider"
There was already missing tests before you start